desktop/windows/install.md

```diff
@@ -124,22 +124,24 @@ Log out and log back in for the changes to take effect.
 
 ### Install from the command line
 
-After downloading **Docker Desktop Installer.exe**, run the following command in a terminal to install Docker Desktop:
+After downloading **Docker Desktop Installer.exe**, open a terminal, and change
+to the directory in which you downloaded the installer, then run the following
+command to install Docker Desktop:
 
-```
-"Docker Desktop Installer.exe" install
+```console
+C:\> "Docker Desktop Installer.exe" install
```

Comment:
Ah here, I don’t think we should prefix the command with any shell prompt, as it seems to imply it’s the Windows Command prompt whereas it’s more a «here is the command to run in a generic manner». Then the two next examples are for specific terminals.

Reply:
Also it doesn’t really have to at the root of the C:, it could be in any folder.

Reply:
The highlighter expects a prompt to be there (and we use prompts in all our examples), but yeah, it's tricky for powershell and command.exe, as "what to use" as prompt (as it defaults to have the path in there); on Linux we just omit that part (`$`), but what should we use for `command.exe` examples (or generic examples for Windows?)

Reply:
I see. Then I suppose `C:\>` is fine for Windows Command prompt examples, at least users are probably familiar enough with it that it should be clear enough what this means.